cypress/e2e/ideyalabs/eholdings/G_eholdings.cy.js

```diff
@@ -30,6 +30,8 @@ describe.skip('Create a custom package', () => {
       eHolding.verifyPackage();
     }
```

Comment:
Fix linter isuue

Reply:
Done in https://github.com/folio-org/stripes-testing/pull/1453